src/coprocessor/dag/expr/fncall.rs

```diff
@@ -176,6 +178,8 @@ impl FnCall {
             ScalarFuncSig::IfTime |
             ScalarFuncSig::IfDuration => (3, 3),
 
+            ScalarFuncSig::JsonArraySig | ScalarFuncSig::JsonObjectSig => (0, usize::MAX),
```

Comment:
It seems inconsistent with `tidb` https://github.com/pingcap/tidb/blob/master/expression/builtin.go#L958

Reply:
It's a TiDB bug. I will fix that.